src/reanimated2/commonTypes.ts

```diff
@@ -94,30 +94,92 @@ export type MapperRegistry = {
   stop: (mapperID: number) => void;
 };
 
-export type Context = Record<string, unknown>;
+export type WorkletClosure = Record<string, unknown>;
 
-export interface WorkletFunction {
-  _closure?: Context;
+interface ReleaseInitData {
+  code: string;
+}
+
+interface DevInitData {
+  code: string;
```

Comment:
```suggestion
interface DevInitData extends ReleaseInitData {
```

Reply:
I did it on purpose. Since it's only one property that would be inherited I decided to not make dependencies here yet and have a rather clear distinction between the two.